GenderPayGap.Database/Migrations/20200625092812_AddDraftReturnTable.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GenderPayGap.Database.Migrations
+{
+    public partial class AddDraftReturnTable : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "DraftReturns",
+                columns: table => new
+                {
+                    DraftReturnId = table.Column<long>(nullable: false)
+                        .Annotation("SqlServer:ValueGenerationStrategy", SqlServerValueGenerationStrategy.IdentityColumn),

```

Comment:
Is it going to work after migration?

Reply:
Probably not - I think we'll have to do another round of "squash all the database migrations"